python/cudf/io/csv.py

```diff
@@ -20,10 +20,13 @@ def _wrap_string(text):
 
 def read_csv(filepath, lineterminator='\n',
              quotechar='"', quoting=True, doublequote=True,
+             windowslinetermination=False, header=-1,
```

Comment:
Does `header=-1` mean don't read column names from the header line? If so the default parameter should be `header=None`.